homeassistant/components/tessie/services.yaml

```diff
@@ -0,0 +1,30 @@
+share:
+  fields:
+    device_id:
+      required: true
+      selector:
+        device:
+          filter:
+            integration: tessie
+    value:
+      required: true
+      selector:
+        text:
+    locale:
+      required: false
+      default: "en-US"
+      selector:
+        text:
+    retry_duration:
+      required: false
+      default: 40
+      selector:
+        number:
+          min: 0
+          max: 90
+          unit_of_measurement: seconds
+    wait_for_completion:
+      required: false
+      default: true
+      selector:
+        boolean:
```

Comment:
why do we ask for locale, retry duration and completion? We should pick something that would work for everyone

Reply:
Why? Can't we pick something that works for everyone? As in, can you elaborate?

Reply:
`wait_for_completion` dictates if the web service returns immediately, or waits for the car to ack the command. You could compare this to calling the `script.my_script` action, vs calling `script.turn_on: script.my_script`. It always depends what the user intends to do (async vs sync call).

`retry_duration` is a courtesy of Tessie: they will retry the call for you in case of timeout. Again, it depends what the user's willing.

`locale` will define how Tesla (not Tessie) will try to parse the address/coordinates (comma vs dot, I guess, etc) before sending it to the vehicle. It has no impact on video URLs. From Tessie's doc:
> The language and country code locale of the address. Useful for helping addresses translate to the navigation system accurately.